apps/rotm/controllers/confirm.js

```diff
@@ -0,0 +1,7 @@
+'use strict';
+
+const BaseConfirmController = require('hof').controllers.confirm;
+
+module.exports = class ConfirmController extends BaseConfirmController {
+
+};
```

Comment:
Just a question, is this needed? or could you just do `controller: require('hof').controllers.confirm` in the index.js?